### PR TITLE
Add new php-based UnitDB  (fixes #16)

### PR DESCRIPTION
**Pull request by Brutus5000**
- [x] tested on test server
- [ ] fix php timeout issue

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -1,6 +1,12 @@
 #
 # Users that do not exist in the respective Docker image must be specified by their user ID
 #
+
+# System user & group
+FAF_FS_PUID=1003
+FAF_FS_PGID=1000
```
Comment (micheljung) [inline .env.template]: These are UIDs on the host system, faf-stack should not make any assumptions.
Keep them `root`

File: config.template/faf-unitdb/datafiles.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[
+	"DATA/GAMEDATA/units.scd.3599",
+	"DATA/GAMEDATA/projectiles.scd.3599"
+]
```
Comment (micheljung) [inline config.template/faf-unitdb/datafiles.json]: Seeing that there are version numbers in the files, will this work out of the box when I clone faf-stack, and copy this config over? Or will the application throw a file not found error at me, cause me time to investigate the issue only to find out that the default is misconfigured?

If so, I suggest commenting-out this (at least the content of the array) so it's clear that it's just for reference. Not sure how the application behaves when we do.

File: config.template/faf-unitdb/locfiles.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[	
+	"DATA/LOC/loc_US.scd.3599",
+	"DATA/LOC/loc.nx2",
+	"DATA/LOC/loc.nmd"
```
Comment (micheljung) [inline config.template/faf-unitdb/locfiles.json]: Same

**Review by micheljung — CHANGES_REQUESTED** [review]

**Review by micheljung — APPROVED** [review]

File: config.template/faf-unitdb/datafiles.json (hunk 2 above)
Comment (Brutus5000) [reply config.template/faf-unitdb/datafiles.json:4]: No this is the biggest problem with this particular unitdb.
It requires version-specific files and they also need to be in the order of the patches.

If you don't provide any the unitdb won't work.
The 3599-files should be the retail files afaik.

**Pull request merged**